### PR TITLE
Self-nomination for Severin Neumann in the 2022 GC election

### PR DESCRIPTION
**Pull request by svrnm**

### CODE REVIEW

File: elections/2022/governance-committee-candidates.md [hunk 1]
```diff
@@ -106,3 +107,51 @@ I would love to help shape how our end users would get their biggest value from
...
+Being passionate about a happy community, my own mission so far is to improve
+the ease of use of OpenTelemetry via my contributions
+(e.g. by [writing documentation](https://github.com/open-telemetry/opentelemetry.io/issues?q=author%3Asvrnm))
+and to encourage as many people as possible within Cisco to contribute to
+the OpenTelemetry projects: [I support 60 individuals](https://opentelemetry.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=Cisco)
+on their journey to from their first issue/PR to being an active member
+of the community.
+
+I would be honored to be elected by you into the Governance Committee and
+to expand my mission to the whole project. If elected, ...
```
Comment (jpkrohling) [inline elections/2022/governance-committee-candidates.md:147]: Looks like this is missing something.

Comment (svrnm) [reply elections/2022/governance-committee-candidates.md:147]: Thanks, there was nothing missing, it was meant to so If elected ... <1,2,3 what you can read in the bullet points>, but it's not clear indeed, so I added ellipses to the bullet points as well

**Comment by SergeyKanzhelev** [comment]
+1

**Comment by yurishkuro** [comment]
+1

**Review by lizthegrey — APPROVED** [review]
Three ratifications, approving

**Pull request merged**